pkg/kubelet/kubelet_node_status.go

```diff
@@ -165,7 +165,7 @@ func (kl *Kubelet) reconcileHugePageResource(initialNode, existingNode *v1.Node)
 		if !supportedHugePageResources.Has(string(resourceName)) {
 			delete(existingNode.Status.Capacity, resourceName)
 			delete(existingNode.Status.Allocatable, resourceName)
-			klog.Infof("Removing now unsupported huge page resource named: %s", resourceName)
+			klog.Infof("Huge page size resource named %s is no longer supported,removing resource from node", resourceName)
```

Comment:
space after comma: supported, removing
But why make this message simpler, like: "Huge page %s is no longer supported"
btw, is there any log analyzers which rely on such messages?

Reply:
My suggestion would be, if we're changing this log message, to migrate the whole file to [structured logging](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md). We will need a release note in either case.

/hold

Reply:
> My suggestion would be, if we're changing this log message, to migrate the whole file to [structured logging](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md). We will need a release note in either case.
> 
> /hold

Hello,I fixed the  whole file(kubelet_node_status.go) to structured logging,but it seems something problem.
Modified thousands of files?
How can i do it？

Reply:
> My suggestion would be, if we're changing this log message, to migrate the whole file to [structured logging](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md). We will need a release note in either case.
> 
> /hold

I have fixed it.
Can you review it for me?
Thanks.
@ehashman